sdk/typescript/src/graphql/generated/2024.1/schema.graphql

```diff
@@ -3591,7 +3591,7 @@ type SystemParameters {
 	"""
 	validatorLowStakeThreshold: BigInt
 	"""
-	Validators with stake below this threshold will be removed from the the active validator set
```

Comment:
This is auto generated I think, so it needs to be fixed in the code. Please remove this file from this PR so that we can merge the rest! Thanks